src/pages/ALLO_ManageAllocations.page

```diff
@@ -71,10 +71,10 @@
                 $('#totalAmount').text(reformatCurrency(unallocated));
                 if (unallocated < 0) {
                     $('#totalAmount').css({ "color": "red", "font-weight": "bold" });
-                    $('[id$="saveCloseBTN"]').attr('disabled','disabled');
```

Comment:
I'm not seeing the buttons enable/disable on the page. I think it's because the Id on the button component is an Apex attribute id, creating a fully qualified Id along the lines of `j_id0:theForm:j_id6:j_id7:saveBTN`. The selector here would need to either select that fully formed Id (challenging) or use a class selector - maybe something like `$('.npsp-pageheader-savebtn')`

Reply:
@force2b I did test this out and the button should be disabling now.

Reply:
I'm not seeing that behavior in my scratch org. Also, do you think we fix the error message styling on this page in this branch?